lib/OrdinaryDiffEqFIRK/src/firk_caches.jl

```diff
@@ -587,12 +589,15 @@ function alg_cache(alg::AdaptiveRadau, u, rate_prototype, ::Type{uEltypeNoUnits}
         z[i] = w[i] = zero(u)
     end
 
+    c_prime = Vector{BigFloat}(undef, num_stages) #time stepping
```

Comment:
`Vector{typeof(t)}`

Reply:
What would I define ` c_prime::Vector{BigFloat}` as (in the cache itself)?

Reply:
what do you mean?

Reply:
oh, you probably want to introduce `TType` as a type parameter in the cache.